src/shims/unix/fd.rs

```diff
@@ -169,10 +169,12 @@ impl FileDescriptor {
         FileDescriptor(Rc::new(RefCell::new(Box::new(fd))))
     }
 
-    pub fn close<'ctx>(self, communicate_allowed: bool) -> InterpResult<'ctx, io::Result<i32>> {
+    // Destroy this `Rc` using `into_inner` so we can call `close` instead of
+    // implicitly running the destructor of the file description.
```

Comment:
The comment was meant to go inside the function, not outside it.

Github has a handy button to apply these suggestions so you don;t have to copy-paste them. :)